internal/config/benchmark.go

```diff
@@ -80,14 +82,16 @@ type InsertConfig struct {
 
 // UpdateConfig defines the desired state of update config
 type UpdateConfig struct {
-	SkipStrictExistCheck bool   `json:"skip_strict_exist_check,omitempty"`
-	Timestamp            string `json:"timestamp,omitempty"`
+	SkipStrictExistCheck  bool   `json:"skip_strict_exist_check,omitempty"`
+	Timestamp             string `json:"timestamp,omitempty"`
+	DisableBalancedUpdate bool   `json:"disable_balanced_update,omitempty"`
 }
 
 // UpsertConfig defines the desired state of upsert config
 type UpsertConfig struct {
-	SkipStrictExistCheck bool   `json:"skip_strict_exist_check,omitempty"`
-	Timestamp            string `json:"timestamp,omitempty"`
+	SkipStrictExistCheck  bool   `json:"skip_strict_exist_check,omitempty"`
+	Timestamp             string `json:"timestamp,omitempty"`
+	DisableBalancedUpdate bool   `json:"disable_balanced_update,omitempty"`
```

Comment:
I am not sure, is it ok to implement the Bind method to this config object (UpdateConfig, UpsertConfig)?

Reply:
I'll check it and fix it if needed.